src/macports1.0/reclaim.tcl

```diff
@@ -106,28 +107,77 @@ namespace eval reclaim {
     }
 
     proc remove_builds {} {
-        # Delete portdbpath/build
+        # Delete portdbpath/build directories
         global macports::portdbpath
 
-        # The root build folder location
+        # The root build directory location
         set root_build      [file join ${macports::portdbpath} build]
 
         ui_msg "$macports::ui_prefix Build location: ${root_build}"
 
+        if {[macports::global_option_isset ports_dryrun]} {
+            ui_msg "Deleting... (dry run)"
+            ui_info [msgcat::mc "Skipping deletion of %s (dry run)" $root_build]
```

Comment:
Nitpick: This and line 136 still make it sound like $root_build itself will be deleted.

Reply:
fixed thanks